de-lib/src/main/java/org/iplantc/de/client/models/tool/sharing/ToolPermission.java

```diff
@@ -0,0 +1,19 @@
+package org.iplantc.de.client.models.tool.sharing;
+
+import com.google.web.bindery.autobean.shared.AutoBean;
+
+/**
```

Comment:
I think we agreed it'd be a good habit to start adding documentation to any new interfaces and public methods.